ADR/0013-integration-service-api-contracts.md

```diff
@@ -104,6 +104,48 @@ Example for a run with an error:
 ```
 
 
+#### Tekton Result Schema Validation
+The test output of the Tekton result **HACBS_TEST_OUTPUT** will be validated using the [jsonschema validator package](https://github.com/santhosh-tekuri/jsonschema).
+The schema is configured as follows:
+
+```
+{
+  "$schema": "http://json-schema.org/draft/2020-12/schema#",
+  "type": "object",
+  "properties": {
+    "result": {
+      "type": "string",
+      "enum": ["SUCCESS", "FAILURE", "WARNING", "SKIPPED", "ERROR"]
+    },
+    "namespace": {
+      "type": "string"
+    },
```

Comment:
I assume that default values cannot be specified since they are not relevant for schema verifications, but if so, we can specify the default value here.

Reply:
That's correct, but it's mentioned in the other part of this document